.github/ISSUE_TEMPLATE/1-enhancement.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Enhancement request
+description: Request an enhancement for Margo Specification
```

Comment:
would "to the" work better than "for" here?

Reply:
Yes, fixed.